src/assets/content/dictionary/3-click-rule.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: 3-click Rule
-definition:
-sources: []
+definition: ‘It suggests that a user of a website should be able to find any information within three mouse clicks. Else, they will likely become frustrated and leave the site.’
+sources:
+- sourceurl: https://en.wikipedia.org/wiki/Three-click_rule
```

Comment:
```suggestion
definition: 3-click Rule suggests that a user of a website should be able to find any information within three mouse clicks. Else, they will likely become frustrated and leave the site.
sources:
- sourceurl: https://en.wikipedia.org/wiki/Three-click_rule
```
nitpick: Almost there! I've suggested to remove the extra quotation marks, and add the term into the definition. You can click accept to accept the suggestion, or you can make another commit to update the definition. Afterwards I'll be happy to accept and merge!